spec/lib/administrate/search_spec.rb

```diff
@@ -0,0 +1,62 @@
+require "active_support/core_ext/module"
+require "administrate/search"
+require "administrate/resource_resolver"
+require "spec_helper"
+require "support/constant_helpers"
+
+
+require "spec_helper"
+require "administrate/resource_resolver"
+
+describe Administrate::Search do
+  describe "#scope" do
+    let(:controller_path) { "admin/users" }
+    let(:resource_resolver) { Administrate::ResourceResolver.new(controller_path) }
+    let(:scope) { "active" }
+    let(:query) { "#{scope}:" }
+
+    it 'give us the search scope' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.